chart/values.schema.json

```
@@ -1351,6 +1351,14 @@
                         "type": "string"
                     }
                 },
+                "priorityClassName": {
+                    "description": "Set priority value from the priority class name",
```

```suggestion
                    "description": "Specify priority for worker pods.",
```

This reads better to me?

---

I just suggested it once, but this applies to all of them too.

---

Updated all of them.